queue/abstract.lua

```diff
@@ -547,19 +609,27 @@ function method.start()
     end
 
     local _taken = box.space._queue_taken
+
+    -- Format of the temporary space has been changed.
+    -- So recreate it if needed.
+    if _taken ~= nil and _taken:format()[1]['type'] == 'unsigned' then
```

Comment:
Should be implemented some other way because it will cause a problem when downgrading to a previous version.

Reply:
Done.